bogota-july-2018/javascript-calculator/calculadora/calcAvanzada .js

```diff
@@ -0,0 +1,91 @@
+
+//Variables
+var calc = document.getElementById('cldora');
```

Comment:
whats does mean calc?
is it a element, I susggest calculatorElement or a more representative name.